linetools/spectra/xspectrum1d.py

```diff
@@ -766,13 +767,19 @@ def rebin(self, new_wv, all=False, **kwargs):
         all : bool, optional
           Rebin all spectra in the XSpectrum1D object?
           Set masking='none' to have the resultant spectra all be regsitered, but note
-             that there will still be masking
+             that there will still be masking unless otherwise specified in kwargs
 
         Returns
         -------
         XSpectrum1D of the rebinned spectrum
         """
         from .utils import rebin, collate
+        #
+        if 'masking' in kwargs:
+            masking = kwargs['masking']
+        else:
+            masking = 'edges'
+        #
```

Comment:
Is this necessary? Given the default masking for `XSpectrum1D` (and hence `collate()`), is `masking='edges'` anyways...